docs/src/index.rst

```diff
@@ -0,0 +1,11 @@
+Welcome to metatensor-models's documentation!
+=============================================
+
+This is a collection of atomistic models interfaced with ``metatensor``.
+
+.. toctree::
+   :maxdepth: 1
+
+   quickstart
+   how-to
```

Comment:
I think we should also add our `utils` functions maybe as a subfolder `API` in the docs. Even if they are not essentials for the enduser they are useful if you want to develop something.

Reply:
Utils are mainly for developers, which are not really the target of our documentation. I would add them under how-to -> "add a new model" perhaps?